lib/improver/cli/nbhood_land_and_sea.py

```diff
@@ -161,23 +64,20 @@ def process(cube, mask, radius=None, radii_by_lead_time=None, weights=None,
         mask (iris.cube.Cube):
             A cube containing either a mask of topographic zones over land or
             a land-sea mask.
-        radius (float):
-            The radius in metres of the neighbourhood to apply.
-            Rounded up to convert into integer number of grid points east and
-            north, based on the characteristic spacing at the zero indices of
-            the cube projection-x and y coordinates.
-            Default is None.
-        radii_by_lead_time (list):
-            A list with the radius in metres at [0] and the lead_time at [1]
-            Lead time is a List of lead times or forecast periods, at which
-            the radii within 'radii' are defined. The lead times are expected
-            in hours.
-            Default is None
         weights (iris.cube.Cube):
             A cube containing the weights which are used for collapsing the
             dimension gained through masking. These weights must have been
             created using a land-sea mask.
             Default is None.
+        radius (float or list of float):
```

Comment:
Would it be neater just to make this 'radii', always accepting a list, aligning with leadtimes.
Then you can skip some of the branching below.

Reply:
Fair, it does always accept a list so that needs fixing.

Reply:
Would still need the branch below though to select the first item in the list.